src/sentry/static/sentry/app/components/hovercard.jsx

```diff
@@ -1,49 +1,99 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
+import intersection from 'lodash/intersection';
 
 class Hovercard extends React.Component {
   static propTypes = {
+    displayTimeout: PropTypes.number,
     className: PropTypes.string,
     containerClassName: PropTypes.string,
     header: PropTypes.node,
     body: PropTypes.node,
   };
 
+  static defaultProps = {
+    displayTimeout: 100,
+  };
+
   constructor(...args) {
     super(...args);
     this.state = {
       visible: false,
+      rendered: false,
     };
+    this.hoverWait = null;
+    this.cardElement = null;
   }
 
-  handleToggleHovercard = () => {
-    let {header, body} = this.props;
+  handleToggleOn = () => this.toggleHovercard(true);
+  handleToggleOff = () => this.toggleHovercard(false);
```

Comment:
does `off` need to go thought toggleHoverCard? or could it just be a simple setState({visible:false, rendered:false})? does the other logic in the method help it?

it could make toggleHovercard simpler

Reply:
Nah, I'd prefer for it to add the timeout for the hoverout as well, since it feels a little more natural.